### PR TITLE
Moved profiling to the begining of import flow

### PR DESCRIPTION
**Pull request by anmolsachan**
tendrl-bug-id: Tendrl/commons/issues/1065
bugzilla: 1655420
Signed-off-by: Anmol Sachan <anmol13694@gmail.com>

### CODE REVIEW

**Comment by anmolsachan** [comment]
@shirshendu Yes we need a PR there

File: tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py [hunk 1]
```diff
@@ -36,6 +38,34 @@ def run(self):
...
+                out, err, rc = cmd.run()
+                if not err:
+                    volumes = filter(None, out.split("\n"))
+                    ret_val, err = enable_disable_volume_profiling(
+                        volumes, self.parameters)
+                    if not ret_val:
+                        logger.log(
+                            "error",
+                            NS.publisher_id,
+                            {"message": "Failed to complete profiling "
```
Comment (shtripat) [inline tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: I don't think its job anymore. Should correct the log message.

File: tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py [hunk 2]
```diff
@@ -36,6 +38,34 @@ def run(self):
...
+                             },
+                            job_id=self.parameters['job_id'],
+                            flow_id=self.parameters['flow_id']
+                        )
+                        return False
+                else:
+                    logger.log(
+                        "error",
+                        NS.publisher_id,
+                        {"message": "Failed to gather volumes info for "
```
Comment (shtripat) [inline tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: Failed to fetch volumes info for enable/disable profiling.

File: tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py (hunk 1 above)
Comment (anmolsachan) [reply tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: Will make the change.

File: tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py (hunk 2 above)
Comment (anmolsachan) [reply tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: Will do it.

File: tendrl/commons/flows/import_cluster/gluster_help.py [hunk 3]
```diff
@@ -186,3 +188,79 @@ def import_gluster(parameters):
...
+def enable_disable_volume_profiling(volumes, parameters):
+    try:
+        cluster = NS.tendrl.objects.Cluster(
+            integration_id=NS.tendrl_context.integration_id
+        ).load()
+
+        logger.log(
+            "info",
+            NS.publisher_id,
+            {"message": "Starting profiling job."},
```
Comment (shtripat) [inline tendrl/commons/flows/import_cluster/gluster_help.py]: I feel we can remove this log message at all as line#211 and line#238 put log messages for start or stop of profiling.

**Comment by shtripat** [comment]
Fix the commit message and pep8 issues.

File: tendrl/commons/flows/import_cluster/gluster_help.py (hunk 3 above)
Comment (anmolsachan) [reply tendrl/commons/flows/import_cluster/gluster_help.py]: ack

**Comment by shtripat** [comment]
@anmolsachan Fix the build issues including pep8. Still failing.

**Comment by anmolsachan** [comment]
@shtripat This PR does not need any change w.r.t pep8. I have verified locally. This can be merged

**Comment by GowthamShanmugam** [comment]
Need to merge this, otherwise, upstream profile enable won't happen part of import flow.

**Pull request merged**